source/opt/instruction.h

```diff
@@ -35,6 +35,19 @@ class IRContext;
 class Module;
 class InstructionList;
 
+// Relaxed logcial addressing:
+//
+// In the logical addressing model, pointers cannot be stored or loaded.  This
```

Comment:
copied?  'store' and 'load' have the immediate implication that the pointer is being de-referenced.

Reply:
I do mean "store" and "load".  The value of the the pointer cannot be stored into a variable because we are not allowed to have variables of type pointer to pointer.